tools/deploy_ML_translated_documentation.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2018, 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+# Script for publishing translated ML documentation to staging site.
+
+# Non-travis variables used by this script.
+ML_SOURCE_REPOSITORY="https://github.com/Qiskit/qiskit-machine-learning.git"
+SOURCE_DOC_DIR="docs/_build/html"
+SOURCE_DIR=`pwd`
+
+STABLE_VERSION=`cat ./qiskit_machine_learning/VERSION.txt`
+FORMATED_VERSION=`echo $STABLE_VERSION | cut -d "." -f -2`
+
+curl https://downloads.rclone.org/rclone-current-linux-amd64.deb -o rclone.deb
+sudo apt-get install -y ./rclone.deb
+
+RCLONE_CONFIG_PATH=$(rclone config file | tail -1)
+
+set -e
+
+# Clone the sources files and po files to ml_docs_source/
+git clone $ML_SOURCE_REPOSITORY ml_docs_source
+git checkout stable/$FORMATED_VERSION
+
+rclone sync -v --exclude='locale/**' ml_docs_source/docs docs
```

Comment:
Were you planning on including the release notes in the translated builds? Won't this exclude all the release notes since you're not building in the ml checkout?

Reply:
I have excluded the release notes .po file ([ML deploy script](https://github.com/Qiskit/qiskit-machine-learning/blob/stable/0.2/tools/deploy_translatable_strings.sh#L61)). AFAIK If sphinx doesn't find the corresponding .po file for a file, it will automatically use the English one.

Is that what you were asking?

Reply:
It was a bit more than that. The release notes are compiled from a combination of the git history and the release note yaml files (by default in `releasenotes/notes`) when the `release-notes` sphinx directive is called (which is done here: https://github.com/Qiskit/qiskit-machine-learning/blob/main/docs/release_notes.rst ). If you don't copy the full git history you get an error because it can't figure out which release a note was added in (which I think you hit before). If you don't include the `releasenotes/notes` directory I believe the release notes page will just be empty. So if I'm reading this script correctly when you build the translated docs the release notes page will be blank instead of English (which is what I was expecting).

Reply:
oh, I see what you mean. hmm.. Let me write something up for that.

Reply:
Maybe this might help https://github.com/qiskit-community/qiskit-translations/pull/686/commits/a59c6f44294978d8798903cce0d41a35ac16864e. This is working for the staging site